benches/bench_eq.rs

```diff
@@ -27,5 +44,39 @@ pub fn eq_eval(c: &mut Criterion) {
     });
 }
 
-criterion_group!(bench_eq, eq_keygen, eq_eval);
+pub fn eq_batch_eval(c: &mut Criterion) {
+    let mut keys_a: Vec<u8> = Vec::with_capacity(3105);
```

Comment:
Please add a comment to recap where this magic number comes from (either with the math or with a link to some code that already contains this number)